### PR TITLE
8297693: Fix typos in src/hotspot and test/hotspot files

### PR DESCRIPTION
**Pull request by jaikiran**
Can I please get a review for this change which only fixes typos in src/hotspot and test/hotspot files? These changes were originally done by @mernst in PR https://github.com/openjdk/jdk/pull/10029, but given that the other PR touches multiple other files and areas, the progress was stalled.

This PR introduces only hotspot related typo fixes that Michael had proposed in the other PR plus also includes a review suggestion (in one of these files) that was made by Alexey in the other PR.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8297693](https://bugs.openjdk.org/browse/JDK-8297693): Fix typos in src/hotspot and test/hotspot files

### Reviewers
 * [Kevin Walls](https://openjdk.org/census#kevinw) (@kevinjwalls - Committer) ⚠️ Review applies to [a7594c33](https://git.openjdk.org/jdk/pull/11386/files/a7594c33981717f2492eddc5ef9552c33c92c04a)
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - **Reviewer**)
 * [Serguei Spitsyn](https://openjdk.org/census#sspitsyn) (@sspitsyn - **Reviewer**)

### Contributors
 * Michael Ernst `<mernst@openjdk.org>`

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11386/head:pull/11386` \
`$ git checkout pull/11386`

Update a local copy of the PR: \
`$ git checkout pull/11386` \
`$ git pull https://git.openjdk.org/jdk pull/11386/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11386`

View PR using the GUI difftool: \
`$ git pr show -t 11386`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11386.diff">https://git.openjdk.org/jdk/pull/11386.diff</a>

</details>

### CODE REVIEW

**Review by kevinjwalls — APPROVED** [review]

File: test/hotspot/jtreg/vmTestbase/nsk/share/locks/DeadlockMaker.java [hunk 1]
```diff
@@ -27,8 +27,8 @@
 import nsk.share.Wicket;
 
 /*
- * Class used to create deadlocked threads. It is possible create 2 or more deadlocked thread, also
- * is is possible to specify resource of which type should lock each deadlocked thread
+ * Class used to create deadlocked threads. It is possible to create 2 or more deadlocked thread, also
+ * it is possible to specify resource of which type should lock each deadlocked thread
```
Comment (dholmes-ora) [inline test/hotspot/jtreg/vmTestbase/nsk/share/locks/DeadlockMaker.java:31]: Even with the corrections this comment still makes little sense, but lets not get bogged down trying to improve these old tests. If you feel like it s/thread,/threads,/ and add a full-stop at the end of the comment.

**Review by dholmes-ora — APPROVED** [review]
Looks good and trivial. One further issue spotted around one of the fixes but feel free ignore.

Thanks for taking care of this @jaikiran !

**Comment by mernst** [comment]
@jaikiran Thanks so much for your help with this.

File: test/hotspot/jtreg/vmTestbase/nsk/share/locks/DeadlockMaker.java (hunk 1 above)
Comment (jaikiran) [reply test/hotspot/jtreg/vmTestbase/nsk/share/locks/DeadlockMaker.java]: Done. I've updated the PR to include this suggested text.

**Review by dholmes-ora — APPROVED** [review]
Thanks.

**Review by sspitsyn — APPROVED** [review]
Good.

Thanks,
Serguei

**Comment by jaikiran** [comment]
Thank you everyone for the reviews.

**Pull request closed without merging**